apps/client/src/hoc/withTracking.js

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from "react";
+import Context from "../context";
+import { useMatomo } from "@datapunt/matomo-tracker-react";
+
+const withTracking = (Component) => ({ ...props }) => {
+  const { topic } = useContext(Context);
+  const { trackEvent } = useMatomo();
+
+  const ClickTrackEvent = ({ category, action, name }) => {
```

Comment:
Nitpicking: rename ClickTrackEvent to MatomoTrackEvent because a keyboard or touch event might not be `click` event